sections/combat.tex

```diff
@@ -71,7 +71,7 @@ \subsection*{\hypertarget{Combatterminology}{Combat Terminology}}
 That Unit \textbf{must skip its next Activation}. Remove the Token instead of activating it.
 If the Unit \textbf{attacked or takes any damage} before that time, \textbf{remove the Token}.
 The Token does not prevent Units from performing Retaliation Attacks.\par
-\textbf{\hypertarget{Defend}{Defend}} \includesvg[height=10px]{\svgs/defense.svg} – Units may choose to gain a Defense Token instead of attacking.
+\textbf{\hypertarget{Defend}{Defend}} \includesvg[height=10px]{\svgs/defense.svg} – Units may choose to gain a Defense Token and end the Activation instead of attacking.
```

Comment:
```suggestion
\textbf{\hypertarget{Defend}{Defend}} \includesvg[height=10px]{\svgs/defense.svg} – Units may choose to gain a Defense Token instead of attacking.
Ranged Units cannot move after gaining it.
```
It also makes it clear that non-ranged units can still move before defending. A lot of people have doubts around this.